contrib/Dockerfile

```diff
@@ -7,8 +7,11 @@
 FROM ubuntu:20.04
 LABEL maintainer="Dgraph Labs <contact@dgraph.io>"
 
-RUN apt-get update && \
-  apt-get install -y --no-install-recommends \
+RUN export DEBIAN_FRONTEND=noninteractive && \
+  apt-get update -qq && \
+  apt-get install -yq --no-install-recommends apt-utils && \
```

Comment:
can this `apt-utils` get moved with the line#14 set of install's? I'm guessing it's done this way because the ordering is important here. The `dist-upgrade` on line#13 requires `apt-utils` is my guess. Either way, could we club the install's together?

Reply:
apt get dist-upgrade installs packages as well, so I put it in there.

Reply:
Also, in order to use apt-utils for installing packages, you need to have apt-utils already there.